### PR TITLE
Update ActionMonitor.php

### PR DESCRIPTION
**Pull request by jasonbahl**
This applies the `modifyMeta` callback only to post meta changes and not user meta

### CODE REVIEW

File: src/ActionMonitor/ActionMonitor.php [hunk 1]
```diff
@@ -197,13 +197,9 @@ function monitorActions() {
...
 		// Post meta updates
-		$meta_types = [ 'user', 'post', 'page' ];
-
-		foreach ( $meta_types as $type ) {
-			add_action( "updated_{$type}_meta", [ $this, 'modifyMeta' ], 100, 3 );
-			add_action( "added_{$type}_meta", [ $this, 'modifyMeta' ], 100, 3 );
-			add_action( "deleted_{$type}_meta", [ $this, 'modifyMeta' ], 100, 3 );
-		}
+		add_action( "updated_post_meta", [ $this, 'modifyMeta' ], 100, 3 );
+		add_action( "added_post_meta", [ $this, 'modifyMeta' ], 100, 3 );
```
Comment (TylerBarnes) [inline src/ActionMonitor/ActionMonitor.php:201]: I think this removes page meta updates. I believe there are separate hooks for `post_meta` and `page_meta` and the post one applies to all CPT's but not to pages

**Review by TylerBarnes — CHANGES_REQUESTED** [review]
Just left 1 comment

File: src/ActionMonitor/ActionMonitor.php (hunk 1 above)
Comment (TylerBarnes) [reply src/ActionMonitor/ActionMonitor.php:201]: Well 🤔 maybe I'm wrong actually :p

Comment (TylerBarnes) [reply src/ActionMonitor/ActionMonitor.php:201]: Yeah I'm wrong 😆

**Review by TylerBarnes — APPROVED** [review]
Tested this locally and it seems to be working well 👍

File: src/ActionMonitor/ActionMonitor.php (hunk 1 above)
Comment (jasonbahl) [reply src/ActionMonitor/ActionMonitor.php:201]: ok, I'll be revisiting to add meta tracking for other entities and will make sure pages work when I get there.

**Pull request merged**